docs/en/01-how-to-build/jetsons.md

```diff
@@ -194,6 +194,20 @@ Please install dependencies:
 sudo apt-get install protobuf-compiler libprotoc-dev
 ```
 
+#### Install ONNX Runtime
```

Comment:
If it's jetson devices, most of the cases, users won't use onnx runtime as inference engine. Maybe we could add this step as **optional**

Reply:
OK.